test/octopus_container_test_framework.go

```diff
@@ -43,7 +44,7 @@ type OctopusContainer struct {
 	URI string
 }
 
-type mysqlContainer struct {
+type MysqlContainer struct {
```

Comment:
mysql ?

Reply:
I made that public to return it from the arrange method for cleanup. It's unlikely to be used but I figured I was better off exposing all of the containers/state in the method and cleaning up based off of that rather than some global state.

Reply:
I was actualy referring to the name